service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/LocationMapper.java

```diff
@@ -75,14 +75,26 @@ public Location mapRoleToLocation(POCDMT000002UK01ParticipantRole role) {
 
     public Location mapRecipientToLocation(POCDMT000002UK01IntendedRecipient intendedRecipient) {
```

Comment:
Don't you need to update unit tests?

Reply:
Please review again now, test has been updated